### PR TITLE
Allow the configuration of liveness and readiness times

### PR DESCRIPTION
**Pull request by stefanganzer**
@foxish, @unguiculus
**What this PR does / why we need it**:
Allows the configuration of readiness and liveness parameters. I needed this as I import existing data in the replica set. During indexing it often happens that one of the pods is marked as having failures. It gets deleted, recreated, and then indexing starts again - with the same result of the pod being marked as having failures.

### CODE REVIEW

File: stable/mongodb-replicaset/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: mongodb-replicaset
 home: https://github.com/mongodb/mongo
-version: 3.0.0
+version: 3.0.1
```
Comment (timstoop) [inline stable/mongodb-replicaset/Chart.yaml]: You're adding a feature, please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/mongodb-replicaset/Chart.yaml (hunk 1 above)
Comment (stefanganzer) [reply stable/mongodb-replicaset/Chart.yaml]: Hello, thanks for the suggestion. I pushed a commit updating the version to 3.1.0.

**Comment by unguiculus** [comment]
Never seen such problems. How big are you imports?

**Pull request merged**